mbway/src/main/java/com/adyen/checkout/mbway/internal/ui/view/MbWayView.kt

```diff
@@ -92,10 +91,11 @@ internal class MbWayView @JvmOverloads constructor(
                 onCountrySelected(country)
             }
         }
-        val firstCountry = countries.firstOrNull()
-        if (firstCountry != null) {
-            binding.autoCompleteTextViewCountry.setText(firstCountry.toShortString())
-            onCountrySelected(firstCountry)
+        val initialCountry =
+            countries.firstOrNull { it.isoCode == DefaultMBWayDelegate.ISO_CODE_PORTUGAL } ?: countries.firstOrNull()
```

Comment:
Is this necessary here? Since countries should only have the supported countries right.
```suggestion
        val initialCountry = countries.firstOrNull()
```

Reply:
Yes the list contains Spain and Portugal and the order is not guaranteed. We have to preselect Portugal. Using just `countries.firstOrNull()` could preselect Spain instead

Reply:
I see. I guess it happens when the names are localized right?

My thinking is to move that logic to the delegate.
What if we create a `getDefaultSupportedCountry()` function in the delegate and return the ISO code or the country itself?